decidim_app-design/packages/core/package.json

```diff
@@ -10,8 +10,7 @@
   "author": "Decidim Contributors",
   "license": "AGPL-3.0",
   "dependencies": {
-    "@hotwired/turbo-rails": "^7.2.0",
-    "@picmo/popup-picker": "^5.7.3",
+    "@picmo/popup-picker": "^5.6.1",
```

Comment:
You may have a regression of #10291 here . 
Please make sure you keep @picmo/popup-picker to ^5.7.3.

Reply:
Fixed

Reply:
This does not to be fixed ...